src/dataprotection/azext_dataprotection/generated/commands.py

```diff
@@ -80,20 +80,20 @@ def load_command_table(self, _):
         'dataprotection backup-instance', dataprotection_backup_instance, client_factory=cf_backup_instance
     ) as g:
         g.custom_command('list', 'dataprotection_backup_instance_list')
-        g.custom_show_command('show', 'dataprotection_backup_instance_show')
-        g.custom_command('create', 'dataprotection_backup_instance_create', supports_no_wait=True)
-        g.custom_command('delete', 'dataprotection_backup_instance_delete', supports_no_wait=True, confirmation=True)
+        # g.custom_show_command('show', 'dataprotection_backup_instance_show')
+        # g.custom_command('create', 'dataprotection_backup_instance_create', supports_no_wait=True)
+        # g.custom_command('delete', 'dataprotection_backup_instance_delete', supports_no_wait=True, confirmation=True)
```

Comment:
Can you clean the code

Reply:
As the `backup-instance` subgroup has not fully migrated yet, I have left the migrated commands in as reference. The idea is that when the full migration is completed, this code will be cleaned up properly. Is that alright, or should I just remove these lines?